src/doc/rustc/src/platform-support.md

```diff
@@ -431,6 +431,7 @@ target | std | host | notes
 `x86_64-unknown-l4re-uclibc` | ? |  |
 [`x86_64-unknown-linux-none`](platform-support/x86_64-unknown-linux-none.md) | * |  | 64-bit Linux with no libc
 [`x86_64-unknown-managarm-mlibc`](platform-support/managarm.md) | ? |   | x86_64 Managarm
+`x86_64-unknown-motor` | ? |  | x86_64 Motor OS
```

Comment:
Problem: please include platform support docs for this target.

Reply:
Done, thanks!